templates/emails/collective.expense.missing.receipt.hbs

```diff
@@ -11,7 +11,7 @@ Subject: Action Required: Submit receipt for card expense
 <p>
 {{#if user}}
 A card attached to {{fromCollective.name}} was charged {{currency expense.amount currency=expense.currency}} and we need the responsible admin to input additional
-information.
+information. This expense was submitted to {{collective.name}}.
```

Comment:
Why do we only have this under the `#if user` condition?

Reply:
Looks like this is a remain that was made useless after this refactor https://github.com/opencollective/opencollective-api/pull/6874  and can be removed if we're not using it anymore.

Reply:
Removed the else part. 👍🏽